tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/PublicAssetsCapability.scala

```diff
@@ -0,0 +1,45 @@
+package com.linagora.tmail.james.jmap.method
+
+import com.google.inject.multibindings.Multibinder
+import com.google.inject.{AbstractModule, Inject}
+import com.linagora.tmail.james.jmap.method.CapabilityIdentifier.LINAGORA_PUBLIC_ASSETS
+import org.apache.commons.configuration2.Configuration
+import org.apache.james.jmap.core.CapabilityIdentifier.CapabilityIdentifier
+import org.apache.james.jmap.core.UnsignedInt.UnsignedInt
+import org.apache.james.jmap.core.{Capability, CapabilityFactory, CapabilityProperties, UnsignedInt, UrlPrefixes}
+import play.api.libs.json.{JsObject, Json}
+
+import jakarta.inject.Named
+
+case class PublicAssetTotalByteLimit(value: UnsignedInt)
+
+case class PublicAssetsCapabilityProperties(publicAssetTotalByteSize: PublicAssetTotalByteLimit) extends CapabilityProperties {
+  override def jsonify(): JsObject = Json.obj("publicAssetTotalSize" -> publicAssetTotalByteSize.value.value)
+}
+
+final case class PublicAssetsCapability(properties: PublicAssetsCapabilityProperties,
+                                        identifier: CapabilityIdentifier = LINAGORA_PUBLIC_ASSETS) extends Capability
+object PublicAssetsConfiguration {
+  val PUBLIC_ASSET_TOTAL_BYTE_SIZE_PROPERTY: String = "publicAssetTotalSize"
```

Comment:
Should be documented e.g. in `docs/modules/ROOT/pages/tmail-backend/configure/index.adoc` IMO.